src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3282,31 +3282,13 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
     # print(target_regions)
     from msrestazure.tools import resource_id, is_valid_resource_id
     from azure.cli.core.commands.client_factory import get_subscription_id
-    from azure.cli.core._profile import Profile
 
     ImageVersionPublishingProfile, GalleryArtifactSource, ManagedArtifact, ImageVersion, TargetRegion = cmd.get_models(
         'GalleryImageVersionPublishingProfile', 'GalleryArtifactSource', 'ManagedArtifact', 'GalleryImageVersion',
         'TargetRegion')
     aux_subscriptions = _get_image_version_aux_subscription(managed_image, os_snapshot, data_snapshots)
     client = _compute_client_factory(cmd.cli_ctx, aux_subscriptions=aux_subscriptions)
 
-    # Auxiliary tokens, pass it to init or operation
-    external_bearer_token = None
-    if aux_subscriptions:
-        profile = Profile(cli_ctx=cmd.cli_ctx)
-        resource = cmd.cli_ctx.cloud.endpoints.active_directory_resource_id
-        cred, _, _ = profile.get_login_credentials(resource=resource,
-                                                   aux_subscriptions=aux_subscriptions)
-        _, _, _, external_tokens = cred.get_all_tokens('https://management.azure.com/.default')
```

Comment:
By moving this to core, it now works for sovereign cloud (solves https://github.com/Azure/azure-cli/pull/15750#discussion_r546513080).